packages/babel-preset-sui/lib/index.js

```diff
@@ -3,14 +3,25 @@ const cleanList = require('./clean-list')
 
 module.exports = {
   'presets': cleanList([
-    require('babel-preset-es2015'),
+    ["env", {
+      "debug": false,
+      "targets": {
+        "node": "6.0.0",
+        "browsers": [
+          "> 1%",
```

Comment:
That is the FC browser support matrix, isnt ?!

Reply:
it is. I only added the node field. The rest is the same.